backend/src/services/storage/storage.service.ts

```
@@ -340,38 +350,80 @@ export class StorageService {
+        .map((key) => ({ key, message: DELETE_OBJECT_FAILURE_MESSAGE }));
+      failed.push(...unreconciledFailures);
+
+      if (failed.length > 0) {
+        logger.warn('Storage provider batch delete partially failed after DB rows were deleted', {
+          bucket,
+          failed,
+        });
+      }
+      const failedKeys = new Set(failed.map((failure) => failure.key));
+      return {
+        deleted: dbDeletedKeys.filter((key) => providerDeletedKeys.has(key) && !failedKeys.has(key)),
```

ESLint

[error] Replace `(key)·=>·providerDeletedKeys.has(key)·&&·!failedKeys.has(key)` with `⏎··········(key)·=>·providerDeletedKeys.has(key)·&&·!failedKeys.has(key)⏎········`

(prettier/prettier)